core/tests/webdriverio_utils/ContributorDashboardAdminPage.js

```diff
@@ -177,6 +182,21 @@ var ContributorDashboardAdminPage = function() {
       'Submit Question Right Element taking too long to appear');
     expect(await contributionRights.getText()).toBe('Allowed');
   };
+
+  this.checkstatslistelements = async function(number) {
```

Comment:
name of the function should be camelCase*

Reply:
Maybe call it expectStatsElementCountToBe

Reply:
Updated. Thanks for nit.